app/styles/vars.scss

```diff
@@ -1,3 +1,10 @@
+// Breakpoints
+
+$breakpoint-xs: 360px;
+$breakpoint-s: 640px;
+$breakpoint-m: 768px;
+$breakpoint-l: 1024px;
```

Comment:
@jishaal let me know what you think - it's basically same as tailwind, with a cheeky xs breakpoint just in case we need a slightly different design on very very small pages

Reply:
Look good to me, don't imagine us need any larger rn